anyTV/music-dashboard
# Use /category/tag/auto instead of /category/tag/search

## A. Summary of Changes

### a. DB Changes:
- N/A

### b. Package Changes:
- none

### c. Implementation Changes:
- Use /category/tag/auto instead of /category/tag/search

## B Reference:

### a. Ticket link/Request Feature link
- [Tunes-131](https://freedom.myjetbrains.com/youtrack/issue/Tunes-131)

### b. Tests Done
- Go to Tracks tab
- Click add tag button
- Type your query
- Suggestions should be shown quickly
